src/Resources/translations/validators.fr.yml

```diff
@@ -0,0 +1,23 @@
+setono_sylius_feed:
+    google:
+        shopping:
+            product:
+                id:
+                    length_min: Cette chaîne est trop courte. Elle doit avoir au maximum 1 caractère
+                    length_max: Cette chaîne est trop longue. Elle doit avoir au maximum 50 caractères
+                brand:
+                    not_blank: La marque est videThe brand is blank
```

Comment:
```suggestion
                    not_blank: La marque est vide
```

Reply:
@loevgaard Oh my bad! Gonna fix it right now